scripts/test-dist.js

```diff
@@ -20,7 +20,12 @@ shell.exec("npm init -y", { cwd: tmpDir });
 shell.exec(`npm install "${tarPath}"`, { cwd: tmpDir });
 shell.config.silent = false;
 
-const cmd = `yarn test --color --runInBand --reporters default --reporters jest-junit ${
+const reporters =
+  +process.version.match(/^v\d+\./)[1] >= 6
```

Comment:
As far as I can tell this condition is always false. I think you forgot parentheses around `\d+`:

```js
+process.version.match(/^v(\d+)\./)[1] >= 6
```

Reply:
Or `require('semver').major(process.version) >= 6`